go/src/koding/klientctl/endpoint/machine/info.go

```diff
@@ -0,0 +1,124 @@
+package machine
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"koding/klient/machine"
+)
+
+// Info stores the basic information about the machine.
+type Info struct {
+	// ID is an unique identifier for a given machine.
+	ID string `json:"id"`
+
+	// Alias stores a human readable alias for machine.
+	Alias string `json:"alias"`
+
+	// Team is the name of the team which created the machine.
+	Team string `json:"team"`
+
+	// Stack describes machine parent stack.
+	Stack string `json:"stack"`
+
+	// Provider represents machine provider.
+	Provider string `json:"provider"`
+
+	// Label is the machine label, as seen by the Koding UI.
+	Label string `json:"label"`
+
+	// The IP of the running machine.
+	IP string `json:"ip"`
```

Comment:
Can we also have `RegisterURL string` field? This field (jMachine.registerUrl) could be used for building klient endpoints (`(*kite.Client).URL`) instead of the IP address. Would be helpful when path-routing for tunnelserver is implemented.